lib/biokbase/catalog/db.py

```diff
@@ -1337,49 +1459,52 @@ def update_db_1_to_2(self):
 
             # make sure everything has the dynamic service flag
             if not 'dynamic_service' in m['info']:
-                self.modules.update_one(
+                modules_collection.update_one(
                     {'_id': m['_id']},
                     {'$set': {'info.dynamic_service': 0}})
 
             if m['current_versions']['release']:
                 if not 'dynamic_service' in m['current_versions']['release']:
-                    self.modules.update_one(
+                    modules_collection.update_one(
                         {'_id': m['_id']},
                         {'$set': {'current_versions.release.dynamic_service': 0}})
 
             if m['current_versions']['beta']:
                 if not 'dynamic_service' in m['current_versions']['beta']:
-                    self.modules.update_one(
+                    modules_collection.update_one(
                         {'_id': m['_id']},
                         {'$set': {'current_versions.beta.dynamic_service': 0}})
 
             if m['current_versions']['dev']:
                 if not 'dynamic_service' in m['current_versions']['dev']:
-                    self.modules.update_one(
+                    modules_collection.update_one(
                         {'_id': m['_id']},
                         {'$set': {'current_versions.dev.dynamic_service': 0}})
 
         # also ensure the execution stats fields have correct names
-        self.exec_stats_apps.update_many({'avg_queue_time': {'$exists': True}},
+        exec_stats_apps_colleciton = db[MongoCatalogDBI._EXEC_STATS_APPS]
+        exec_stats_apps_colleciton.update_many({'avg_queue_time': {'$exists': True}},
```

Comment:
```suggestion
        exec_stats_apps_collection = db[MongoCatalogDBI._EXEC_STATS_APPS]
        exec_stats_apps_collection.update_many({'avg_queue_time': {'$exists': True}},
```

Reply:
Is this conversation the same as [this](https://github.com/kbase/catalog/pull/140#discussion_r1975967554)? Also, the suggested change you mentioned doesn't indicate anything different.

Reply:
I'm not sure what you're talking about, this is just a spelling correction

Reply:
fixed!

Reply:
:+1: